### PR TITLE
Add GameObject, Renderer and change RenderWindow placement

### PR DESCRIPTION
**Pull request by kicerKs**
Moved drawing to new Renderer class
Changed RenderWindow placement to class instead of gameloop
Added GameObject class
Added gameObjects vector to store all GameObjects and also added for loop in update function (gameloop)

### CODE REVIEW

**Comment by kicerKs** [comment]
[GameObject](https://trello.com/c/kdE5Yjkc/8-gameobject)

**Comment by kicerKs** [comment]
[Renderer](https://trello.com/c/iyqAXogi/14-renderer)

File: poketechnika/GameScene.cpp [hunk 1]
```diff
@@ -15,5 +15,5 @@ void GameScene::loadTextures()
 }
 
 void GameScene::draw(sf::RenderWindow& window) {
```
Comment (karofox) [inline poketechnika/GameScene.cpp]: does this method still need `sf::RenderWindow& window`?

File: poketechnika/MenuScene.cpp [hunk 2]
```diff
@@ -15,5 +15,5 @@ void MenuScene::loadTextures()
 }
 
 void MenuScene::draw(sf::RenderWindow& window) {
```
Comment (karofox) [inline poketechnika/MenuScene.cpp]: does this method still need `sf::RenderWindow& window`?

**Review by karofox — APPROVED** [review]

**Pull request merged**